draft-ietf-oauth-transaction-tokens.md

```diff
@@ -433,6 +437,19 @@ grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Atoken-exchange
 ~~~
 {: #figtxtokenrequest title="Example: Txn-Token Request"}
 
+### Subject Token Types
+The `subject_token_type` parameter in the Txn-Token Request MAY have one of the following values:
+
+rfc9068
+: The `subject_token` is a JWT format OAuth 2.0 Access Token as defined in {{RFC9068}}
+
+id-token
+: The `subject_token` is an OpenID Connect {{OpenIdConnect}} ID Token
+
+self-signed
+: The `subject_token` is a self-signed JWT issued by the requesting workload
+
+The Txn-Token Service MAY support other formats, which MAY be specified in the `subject_token_type` parameter
```

Comment:
Maybe add: "Note, as defined in RFC8693, the `subject_token_type` must be a URI.

Reply:
True, and it means the current document should have an IANA subsection to define it.